internal/venice-common/src/main/java/com/linkedin/venice/meta/QueryAction.java

```diff
@@ -16,5 +16,8 @@ public enum QueryAction {
   DICTIONARY,
 
   // Admin request from server admin tool
-  ADMIN
+  ADMIN,
+
+  // METADATA is a GET request to /metadata/storename on the storage node to fetch metadata for that node
+  METADATA
```

Comment:
This needs to be added in the exceptions in `getQueryActionFromRequest`. Probably better to remove the hard coding there by printing all the possible enum strings in the exception?

Reply:
added a variable to that exception string in `getQueryActionFromRequest`. 

Follow up: wondering if this variable could just be a static field of the class so we're not regenerating it every time